Node.Dockerfile

```
@@ -0,0 +1,10 @@
+FROM node:11
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends apt-utils dialog 2>&1 \
+    && apt-get -y install git procps lsb-release
+RUN apt-get autoremove -y \
+    && apt-get clean -y \
+    && rm -rf /var/lib/apt/lists/*
+ENV DEBIAN_FRONTEND=
+RUN npm install -g nodemon
```

@Netaum, quando você cria um dockerfile sem definir um USER, o docker é criado com um usário ROOT deixando o docker vuneravel.
Acredito ser interessante criar um usuário para aplicação
utilizando a [sugestão](https://github.com/idvogados/backend/pull/16/files#r419056296) do @rodrigondec, poderiamos fazer da seguinte forma:

```suggestion
RUN mkdir /app
WORKDIR /app
COPY . /app
...
RUN chown -R node:node /app
USER node
RUN npm install -g nodemon
```

---

o `RUN npm instal -g nodemon` tem que ser executado antes do `USER node`, se não irá dar problema de permissão

---

A primeira versão tinha o usuário separado. Mas isso é um docker de dev, uma extensão da máquina de dev. Esse container não irá ir pra prod, por isso removi esse usuário separado.

---

Concordo. Podemos simplificar.

Porém acho interessante deixar as configurações do `workdir` no Dockerfile mesmo

---

Concordo. Vou colocar o workdir no container.